resin/models/device.py

```diff
@@ -737,3 +741,81 @@ def get_status(self, uuid):
             return DeviceStatus.CONFIGURING
 
         return DeviceStatus.IDLE
+
+    def set_custom_location(self, uuid, location):
+        """
+        Set a custom location for a device.
+
+        Args:
+            uuid (str): device uuid.
+            location (dict): device custom location, format: { 'latitude': <latitude>, 'longitude': <longtitude> }.
+
+        Raises:
+            DeviceNotFound: if device couldn't be found.
+
+        Examples:
+            >>> location = {
+                'latitude': '21.032777',
+                'longtitude': '105.831586'
+            }
+            >>> resin.models.device.set_custom_location('df09262c283b1dc1462d0e82caa7a88e52588b8c5d7475dd22210edec1c50a',location)
+            OK
+
+        """
+
+        if not self.has(uuid):
+            raise exceptions.DeviceNotFound(uuid)
+
+        params = {
+            'filter': 'uuid',
+            'eq': uuid
+        }
+        data = {
+            'custom_latitude': location['latitude'],
+            'custom_longitude': location['longtitude']
```

Comment:
There's a breaking typo in this param name here, in the example code above, and `unset` below, but in not the parameter docs. `longtitude` should be `longitude`.